src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxIPv6InterfaceProperties.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace System.Net.NetworkInformation
+{
+    internal class LinuxIPv6InterfaceProperties : IPv6InterfaceProperties
+    {
+        private LinuxNetworkInterface _linuxNetworkInterface;
+
+        public LinuxIPv6InterfaceProperties(LinuxNetworkInterface linuxNetworkInterface)
+        {
+            _linuxNetworkInterface = linuxNetworkInterface;
+        }
+
+        public override int Index
+        {
+            get
+            {
+                // We could call if_nametoindex(name), if we wanted.
```

Comment:
I think this is just leftover from when I was only storing the name of the interface, rather than a reference to the whole LinuxNetworkInterface object (which has the index stored in it). I'll remove this comment.